public/apps/configuration/panels/tenant-list/tenant-list.tsx

```diff
@@ -130,24 +134,32 @@ export function TenantList(props: AppDependencies) {
     }
   };
 
-  const viewDashboard = async (tenantName: string) => {
+  const viewDashboard = async (tenantValue: string, action: string) => {
```

Comment:
It is a little bit weird the function name is still 'view' dashboard. Same comments apply to 'view' visualization

Reply:
changed the function name to `viewOrCreateDashboard` and `viewOrCreateVisualization`